plugin/files/airconsole/runtime.js

```diff
@@ -77,7 +77,9 @@ cr.plugins_.AirConsole = function(runtime)
           self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnMessageKey, self);
         }
         if (data.message) {
-          self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnMessage, self);
+          self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnMessageIs, self);
+		  self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnMessageFrom, self);
+		  self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnMessage, self);
```

Comment:
When a message is received I either want to know:
* From which device and which message-data it contains (prev. OnMessage)
* Any device and which message it contains (OnMessageKey)

Like this we trigger OnMessage always.

Also, wouldn't you check anyways what the message-data is if you know that a specific device sent it?
(which in the end is OnMessage). Can you give a use-case?

(Please also check the indentation)